examples/chart/teleport-cluster/values.yaml

```diff
@@ -76,7 +76,10 @@ acmeURI: ""
 # kubectl -n ${TELEPORT_NAMESPACE?} create secret generic license --from-file=/path/to/downloaded/license.pem
 enterprise: false
 
-installCRDs: false
+# CRDs are installed by default when the operator is enabled. This manual override allows to disable CRD installation
+# when deploying multiple releases in the same cluster.
+# installCRDs:
```

Comment:
Just curious: If the InitialCRDs value was removed form chart values does the  https://github.com/gravitational/teleport/blob/hugo/operator-install-crds-by-default/examples/chart/teleport-cluster/.lint/operator.yaml#L4 lint test should fail or it is is valid to pass no-existed path values ?

Reply:
By default Helm will happily accept any value, even if it was not present in the chart's original `values.yaml`. This is a frequent error cause (for example I make a typo or indent badly a field, helm will happily ignore the value and destroy my prod).

There is an optional validation mechanism if the chart maintainer provides a JSON Schema. We are using it and declaring `installCRDs`: https://github.com/gravitational/teleport/blob/b3fd61790b9f935730b3012fea5dd5cdf6b65c0d/examples/chart/teleport-cluster/values.schema.json#L169-L173

This is why the linter is not complaining.